WordPress/Classes/ViewRelated/Stats/Extensions/UITableViewCell+Stats.swift

```diff
@@ -72,3 +72,87 @@ extension UITableViewCell {
     }
 
 }
+
+ extension UITableViewCell {
+    struct StatsTotalRowConfiguration {
+        let limitRowsDisplayed: Bool
+        let rowDelegate: StatsTotalRowDelegate?
+        let referrerDelegate: StatsTotalRowReferrerDelegate?
+        let viewMoreDelegate: ViewMoreRowDelegate?
+    }
+
+    func addDefaultTotalRows(toStackView rowsStackView: UIStackView) {
```

Comment:
Let's move it outside of the `UITableViewCell` extension since it's specific to state.